lib/features/login/presentation/login_controller.dart

```diff
@@ -555,6 +576,10 @@ class LoginController extends ReloadableController {
   }
 
   void _handleCommonOIDCFailure() {
+    if (PlatformInfo.isMobile) {
+      removeCompanyServerLoginInfo();
+    }
```

Comment:
why we need to remove company server here?

Reply:
we should delete when user enters server url, not through dns